ipatests/test_integration/test_backup_and_restore.py

```diff
@@ -126,8 +126,7 @@ def backup(host):
 
     # Get the backup location from the command's output
     for line in result.stderr_text.splitlines():
-        prefix = ('ipa.ipaserver.install.ipa_backup.Backup: '
-                  'INFO: Backed up to ')
+        prefix = ('ipaserver.install.ipa_backup: INFO: Backed up to ')
```

Comment:
Please drop the parenthesis, they are not necessary. Pylint / pep8 should warn about them, too. Did we disable the checker?

Reply:
Thanks, done.